app/src/main/java/com/pose/move/feature/auth/login/LoginViewModel.kt

```diff
@@ -14,7 +14,7 @@ class LoginViewModel @Inject constructor(
 
     fun login(email: String, password: String) {
         viewModelScope.launch {
-            internalStorageManager.setIsUserLoggedIn(true)
+            internalStorageManager.setAuthToken("")
```

Comment:
Shouldn't you set a non-empty value here? Right now you're setting it to empty string when you login and when you clear the storage. This way you can't tell if the user is logged in or not, even if it's just for testing purposes

Reply:
Correct, this is WIP code and it should have been marked with a TODO